3-JSS/js-styles/style.js

```diff
@@ -0,0 +1,96 @@
+import jss from "jss";
+import preset from "jss-preset-default";
+
+jss.setup(preset());
```

Comment:
I suggest separating the setup and style definitions into different files

Reply:
Do you mean to make a file only for the imports and the setup and then import this file into the styling file?